cwms-data-api/src/test/java/fixtures/CwmsDataApiSetupCallback.java

```diff
@@ -144,6 +145,7 @@ public void beforeAll(ExtensionContext context) throws Exception {
 
             // OIDC properties
             System.setProperty("cwms.dataapi.access.providers","KeyAccessManager,OpenID,CwmsAccessManager");
+            System.getProperty(OpenIDAccessManager.CREATE_USERS_KEY,":true");
             System.setProperty(OpenIDAccessManagerProvider.WELL_KNOWN_PROPERTY,KeyCloakExtension.getOidcWellKnown());
```

Comment:
getProperty line here doesn't look right.  Is this supposed to be setProperty?  Is the value supposed to be ":true"?

Reply:
Yep, now it's fixed. also wasn't supposed to have the `:` in there either.